components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -48,11 +45,21 @@ private const val DEFAULT_SERVER_URL = "https://crash-reports.mozilla.com/submit
 class MozillaSocorroService(
     private val applicationContext: Context,
     private val appName: String,
-    private val serverUrl: String = DEFAULT_SERVER_URL
+    private val appId: String = BuildConfig.MOZ_APP_ID,
+    private val version: String = BuildConfig.MOZILLA_VERSION,
+    private val buildId: String = BuildConfig.MOZ_APP_BUILDID,
+    private val vendor: String = BuildConfig.MOZ_APP_VENDOR,
+    private var serverUrlString: String? = null
```

Comment:
nit: Adding the type (`String`) to the name feels wrong. :)

Reply:
I'll remove the `String` portion of the name.